Xamarin.Essentials/Launcher/Launcher.android.cs

```diff
@@ -57,6 +57,9 @@ static Task PlatformOpenAsync(OpenFileRequest request)
             return Task.CompletedTask;
         }
 
+        static Task PlatformOpenAsync(OpenFileRequest request, bool openInApp) =>
```

Comment:
I think we would just add this to "OpenFileReqauest", no need to chagne teh singautre at all